wrench/src/wrench.rs

```diff
@@ -174,39 +179,50 @@ impl Wrench {
         let api = sender.create_api();
 
         let (timing_sender, timing_receiver) = chase_lev::deque();
-        let notifier = Box::new(Notifier::new(window.create_window_proxy(), timing_receiver));
+        let notifier = Box::new(Notifier::new(window.create_window_proxy(), timing_receiver, verbose));
         renderer.set_render_notifier(notifier);
 
+        let gl_version = gl::get_string(gl::VERSION);
+        let gl_renderer = gl::get_string(gl::RENDERER);
+
         let mut wrench = Wrench {
             window_size: size,
 
             renderer: renderer,
             sender: sender,
             api: api,
+            window_title_to_set: None,
 
             rebuild_display_lists: do_rebuild,
+            verbose: verbose,
             device_pixel_ratio: dp_ratio,
 
             image_map: HashMap::new(),
 
             root_pipeline_id: PipelineId(0, 0),
             next_scroll_layer_id: 0,
 
-            //gl_renderer: gl_renderer,
-            //gl_version: gl_version,
-
+            gl_renderer: gl_renderer,
+            gl_version: gl_version,
             frame_start_sender: timing_sender,
         };
 
         wrench.set_title("start");
+        // there's a "frame 0" that webrender itself renders; push this to
+        // not confuse our notifier
+        wrench.frame_start_sender.push(time::SteadyTime::now());
```

Comment:
nit: use `wrench.begin_frame()` instead?
perhaps, `frame_start_sender` doesn't even need to be public